concrete-core/src/specification/engines/glwe_ciphertext_vector_trivial_decryption.rs

```diff
@@ -0,0 +1,39 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::{GlweCiphertextVectorEntity, PlaintextVectorEntity};
+
+engine_error! {
+    GlweCiphertextVectorTrivialDecryptionError for GlweCiphertextVectorTrivialDecryptionEngine @
+}
+
+/// A trait for engines trivially decrypting GLWE ciphertext vectors.
+///
+/// # Semantics
+///
+/// This [pure](super#operation-semantics) operation generates a plaintext vector containing the
+/// trivial decryption of the `input` ciphertext vector.
+///
+/// # Formal Definition
```

Comment:
Here I think we could put a similar description to the one I had put for the encryption.